hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -250,13 +250,9 @@
     }
     {
       name: stub_cpu_mode
-      en_run_modes: ["gen_otp_images_mode"]
-      run_opts: ["+stub_cpu=1"]
-    }
-    {
-      name: stub_cpu_mode_with_test_rom
       en_run_modes: ["sw_test_mode_common"]
-      // The tests using this mode only require the ROM init check to succeed.
+      // Note that the real hardware will always have a valid ROM image, and some stub CPU
+      // tests require that the ROM init check succeeds successfully.
       // The example_test_from_rom test is sufficient.
       sw_images: ["//sw/device/tests:example_test_from_rom:0:test_in_rom"]
```

Comment:
A future enhancement for M3 would be to preload the ROM with random data and compute and set the intg portion to the correct value, rather than preloading an actual test image. That would eliminate the extra bazel invocation step.

Reply:
That's actually a good idea - let me look into this.